httpobs/scanner/grader/grade.py

```diff
@@ -22,7 +22,28 @@
     0: 'F'
 }
 
+# See https://wiki.mozilla.org/Security/Standard_Levels for a definition of the risk levels
+# We cannot make an accurate decision on HIGH and MAXIMUM risk likelihood indicators with the current checks,
+# thus the likelihood indicator is currently at best (or worse) MEDIUM
+LIKELIHOOD_INDICATOR_CHART = {
+    'A+': 'LOW',
+    'A': 'LOW',
+    'A-': 'LOW',
+    'B+': 'MEDIUM',
+    'B': 'MEDIUM',
+    'B-': 'MEDIUM',
+    'C+': 'MEDIUM',
+    'C': 'MEDIUM',
+    'C-': 'MEDIUM',
+    'D+': 'MEDIUM',
+    'D': 'MEDIUM',
+    'D-': 'MEDIUM',
+    'F': 'MEDIUM'
```

Comment:
Yes, that's how it's currently graded - though ideally this is up to you if you want to tweak them.
MEDIUM means we do not think failing any of these will directly put you at risk if the site is more impactful. Say, if both observatory and another scanner returns F/MEDIUM likelihood indicator, it is possible that this triggers a higher likelihood and thus potentially a higher risk. But the F alone does not.

Then again, you can tweak to indicate what the scoring means (in case it changes - for ex if having F means you're directly at risk of an attack then it should be HIGH at least)